ui/src/pages/occurrences/occurrences.tsx

```diff
@@ -77,6 +78,7 @@ export const Occurrences = () => {
         showAppliedFilters
         tooltip={translate(STRING.TOOLTIP_OCCURRENCE)}
       >
+        <TaxonFilter />
```

Comment:
When possible to also apply filtering by determination id to species view, we can just add the same control with one line of code like this! I tried, but seems that filtering did not had any effect in species view, I skipped for now.